src/main/scala/AI.scala

```diff
@@ -1,21 +1,21 @@
 class AI {
 
-  def aiPlay: Int ={
+  def aiRPS: String ={
```

Comment:
this method could be simplified to 
def aiRPS: String ={
  scala.util.Random.nextInt(2) match{
    case 0 => aiInput = "Rock"
    case 1 => aiInput = "Paper"
    case 2 => aiInput = "Scissors"
  }
}